components/eamxx/src/physics/nudging/atmosphere_nudging.hpp

```diff
@@ -37,6 +36,9 @@ class Nudging : public AtmosphereProcess
   template <typename S>
   using view_2d = typename KT::template view_2d<S>;
 
+  //template <typename S>
```

Comment:
If you don't need this, let's remove it.

Reply:
removed